cmd/server-mux_test.go

```diff
@@ -109,20 +109,18 @@ func dial(addr string) error {
 
 // Tests initializing listeners.
 func TestInitListeners(t *testing.T) {
-	portTest1 := getFreePort()
-	portTest2 := getFreePort()
```

Comment:
Yes, `getFreePort` is inherently racy and should not be implemented like this. The right way is to write a function that creates a socket bound to a free port and return that. In the meanwhile lets get this change in.